<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1663</width>
    <height>1137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZOIA Librarian</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resources/logo.ico</normaloff>resources/logo.ico</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="left_widget">
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="movable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="tab_ps_2">
         <attribute name="title">
          <string>PatchStorage View</string>
         </attribute>
         <attribute name="toolTip">
          <string>Switch to view all ZOIA patches on PatchStorage</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="widget_6" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QWidget" name="widget_7" native="true">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_7">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QSplitter" name="splitter_2">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>640</width>
                       <height>480</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="contextMenuPolicy">
                      <enum>Qt::DefaultContextMenu</enum>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::NoFrame</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Plain</enum>
                     </property>
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="childrenCollapsible">
                      <bool>false</bool>
                     </property>
                     <widget class="QWidget" name="widget_8" native="true">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout">
                       <property name="leftMargin">
                        <number>0</number>
                       </property>
                       <property name="topMargin">
                        <number>0</number>
                       </property>
                       <property name="rightMargin">
                        <number>9</number>
                       </property>
                       <property name="bottomMargin">
                        <number>0</number>
                       </property>
                       <item>
                        <widget class="QWidget" name="widget" native="true">
                         <layout class="QHBoxLayout" name="horizontalLayout_3">
                          <property name="leftMargin">
                           <number>0</number>
                          </property>
                          <property name="topMargin">
                           <number>0</number>
                          </property>
                          <property name="rightMargin">
                           <number>0</number>
                          </property>
                          <property name="bottomMargin">
                           <number>0</number>
                          </property>
                          <item>
                           <widget class="QLineEdit" name="searchbar_3"/>
                          </item>
                          <item>
                           <widget class="QPushButton" name="search_button_3">
                            <property name="text">
                             <string>Search</string>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </widget>
                       </item>
                       <item>
                        <widget class="QTableWidget" name="table">
                         <property name="frameShadow">
                          <enum>QFrame::Plain</enum>
                         </property>
                         <property name="sizeAdjustPolicy">
                          <enum>QAbstractScrollArea::AdjustToContents</enum>
                         </property>
                         <property name="editTriggers">
                          <set>QAbstractItemView::NoEditTriggers</set>
                         </property>
                         <property name="selectionMode">
                          <enum>QAbstractItemView::NoSelection</enum>
                         </property>
                         <property name="selectionBehavior">
                          <enum>QAbstractItemView::SelectItems</enum>
                         </property>
                         <property name="horizontalScrollMode">
                          <enum>QAbstractItemView::ScrollPerPixel</enum>
                         </property>
                         <property name="rowCount">
                          <number>1</number>
                         </property>
                         <property name="columnCount">
                          <number>5</number>
                         </property>
                         <attribute name="horizontalHeaderStretchLastSection">
                          <bool>true</bool>
                         </attribute>
                         <attribute name="verticalHeaderVisible">
                          <bool>false</bool>
                         </attribute>
                         <row/>
                         <column/>
                         <column/>
                         <column/>
                         <column/>
                         <column/>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                     <widget class="QTextBrowser" name="text_browser">
                      <property name="frameShadow">
                       <enum>QFrame::Plain</enum>
                      </property>
                      <property name="textInteractionFlags">
                       <set>Qt::TextBrowserInteraction</set>
                      </property>
                     </widget>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_ls">
         <attribute name="title">
          <string>Local Storage View</string>
         </attribute>
         <attribute name="toolTip">
          <string>Switch to your locally saved patches</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QWidget" name="widget_9" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="widget_10" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QWidget" name="widget_11" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_9">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QSplitter" name="splitter_3">
                     <property name="minimumSize">
                      <size>
                       <width>640</width>
                       <height>480</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::NoFrame</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Plain</enum>
                     </property>
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="childrenCollapsible">
                      <bool>false</bool>
                     </property>
                     <widget class="QWidget" name="widget_12" native="true">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_2">
                       <property name="leftMargin">
                        <number>0</number>
                       </property>
                       <property name="topMargin">
                        <number>0</number>
                       </property>
                       <property name="rightMargin">
                        <number>9</number>
                       </property>
                       <property name="bottomMargin">
                        <number>0</number>
                       </property>
                       <item>
                        <widget class="QWidget" name="widget_2" native="true">
                         <layout class="QHBoxLayout" name="horizontalLayout_4">
                          <property name="leftMargin">
                           <number>0</number>
                          </property>
                          <property name="topMargin">
                           <number>0</number>
                          </property>
                          <property name="rightMargin">
                           <number>0</number>
                          </property>
                          <property name="bottomMargin">
                           <number>0</number>
                          </property>
                          <item>
                           <widget class="QLineEdit" name="searchbar_4"/>
                          </item>
                          <item>
                           <widget class="QPushButton" name="search_button_4">
                            <property name="text">
                             <string>Search</string>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </widget>
                       </item>
                       <item>
                        <widget class="QTableWidget" name="table_2">
                         <property name="frameShadow">
                          <enum>QFrame::Plain</enum>
                         </property>
                         <property name="sizeAdjustPolicy">
                          <enum>QAbstractScrollArea::AdjustToContents</enum>
                         </property>
                         <property name="editTriggers">
                          <set>QAbstractItemView::NoEditTriggers</set>
                         </property>
                         <property name="selectionMode">
                          <enum>QAbstractItemView::NoSelection</enum>
                         </property>
                         <property name="horizontalScrollMode">
                          <enum>QAbstractItemView::ScrollPerPixel</enum>
                         </property>
                         <property name="rowCount">
                          <number>1</number>
                         </property>
                         <property name="columnCount">
                          <number>6</number>
                         </property>
                         <attribute name="horizontalHeaderStretchLastSection">
                          <bool>true</bool>
                         </attribute>
                         <attribute name="verticalHeaderVisible">
                          <bool>false</bool>
                         </attribute>
                         <row/>
                         <column/>
                         <column/>
                         <column/>
                         <column/>
                         <column/>
                         <column/>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                     <widget class="QTextBrowser" name="text_browser_2">
                      <property name="frameShadow">
                       <enum>QFrame::Plain</enum>
                      </property>
                      <property name="textInteractionFlags">
                       <set>Qt::TextBrowserInteraction</set>
                      </property>
                     </widget>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_sd">
         <attribute name="title">
          <string>SD Card View</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QTableWidget" name="table_3">
            <property name="minimumSize">
             <size>
              <width>1619</width>
              <height>0</height>
             </size>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
            <property name="dragDropOverwriteMode">
             <bool>false</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::InternalMove</enum>
            </property>
            <property name="defaultDropAction">
             <enum>Qt::MoveAction</enum>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="rowCount">
             <number>64</number>
            </property>
            <property name="columnCount">
             <number>1</number>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <column/>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1663</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiel">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSpecify_SD_Card_Location"/>
    <addaction name="actionCheck_For_Updates"/>
    <addaction name="actionReload_PatchStorage_patch_list"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSort">
    <property name="title">
     <string>Sort</string>
    </property>
    <addaction name="actionSort_by_title_A_Z"/>
    <addaction name="actionSort_by_title_Z_A"/>
    <addaction name="actionSort_by_date_new_old"/>
    <addaction name="actionSort_by_date_old_new"/>
    <addaction name="actionSort_by_likes_high_low"/>
    <addaction name="actionSort_by_likes_low_high"/>
    <addaction name="actionSort_by_views_high_low"/>
    <addaction name="actionSort_by_views_low_high"/>
    <addaction name="actionSort_by_downloads_high_low"/>
    <addaction name="actionSort_by_downloads_low_high"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionZOIA_Librarian_Help"/>
   </widget>
   <addaction name="menuFiel"/>
   <addaction name="menuSort"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSpecify_SD_Card_Location">
   <property name="text">
    <string>Specify SD Card Location</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSort_by_title_A_Z">
   <property name="text">
    <string>Sort by title (A-Z)</string>
   </property>
  </action>
  <action name="actionSort_by_title_Z_A">
   <property name="text">
    <string>Sort by title (Z-A)</string>
   </property>
  </action>
  <action name="actionSort_by_date_new_old">
   <property name="text">
    <string>Sort by date (new-old)</string>
   </property>
  </action>
  <action name="actionSort_by_date_old_new">
   <property name="text">
    <string>Sort by date (old-new)</string>
   </property>
  </action>
  <action name="actionSort_by_likes_high_low">
   <property name="text">
    <string>Sort by likes (high-low)</string>
   </property>
  </action>
  <action name="actionSort_by_likes_low_high">
   <property name="text">
    <string>Sort by likes (low-high)</string>
   </property>
  </action>
  <action name="actionSort_by_views_high_low">
   <property name="text">
    <string>Sort by views (high-low)</string>
   </property>
  </action>
  <action name="actionSort_by_views_low_high">
   <property name="text">
    <string>Sort by views (low-high)</string>
   </property>
  </action>
  <action name="actionSort_by_downloads_high_low">
   <property name="text">
    <string>Sort by downloads (high-low)</string>
   </property>
  </action>
  <action name="actionSort_by_downloads_low_high">
   <property name="text">
    <string>Sort by downloads (low-high)</string>
   </property>
  </action>
  <action name="actionCheck_For_Updates">
   <property name="text">
    <string>Check For Patch Updates</string>
   </property>
  </action>
  <action name="actionReload_PatchStorage_patch_list">
   <property name="text">
    <string>Reload PatchStorage Patch List</string>
   </property>
  </action>
  <action name="actionZOIA_Librarian_Help">
   <property name="text">
    <string>ZOIA Librarian Help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
